ci/no-superuser/create-no-superuser-sqlserver.sh

```diff
@@ -1,23 +1,55 @@
 #!/bin/bash
-set -eu
+set -u
 
 # Get container name and password from arguments
 SQL_SERVER_CONTAINER_NAME=$1
 SQL_SERVER_PASSWORD=$2
+MAX_RETRY_COUNT=$3
+RETRY_INTERVAL=$4
+COUNT=0
+
+# A SQL Server container takes a few seconds to start SQL Server process
+# in the container. So, first, we wait ${RETRY_INTERVAL} seconds before
+# we run the `sqlcmd` command in the container.
+echo "Sleep ${RETRY_INTERVAL} seconds to wait for SQL Server start."
+
+while [[ ${COUNT} -lt ${MAX_RETRY_COUNT} ]]
+do
+    sleep ${RETRY_INTERVAL}
+
+    echo "Retry count: ${COUNT}"
+
+    docker exec -t ${SQL_SERVER_CONTAINER_NAME} /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P ${SQL_SERVER_PASSWORD} -d master -Q "SELECT 1"
+    if [[ $? -eq 0 ]]; then
+        break
+    else
+        echo "INFO: sqlcmd command failed. Will retry after ${RETRY_INTERVAL} seconds."
+    fi
+
+    COUNT=$((COUNT + 1))
+
+    if [[ ${COUNT} -eq ${MAX_RETRY_COUNT} ]]; then
+        echo "ERROR: sqlcmd command failed ${MAX_RETRY_COUNT} times. Please check your configuration." >&2
+        exit 1
+    fi
+done
+
+echo "INFO: sqlcmd command succeeded. Continue creating no superuser."
 
 # Create login
-docker exec -t ${SQL_SERVER_CONTAINER_NAME} /opt/mssql-tools/bin/sqlcmd -S localhost -U SA -P ${SQL_SERVER_PASSWORD} -d master -Q "CREATE LOGIN no_superuser WITH PASSWORD = 'no_superuser_password', DEFAULT_DATABASE = master , CHECK_POLICY = OFF, CHECK_EXPIRATION = OFF"
+docker exec -t ${SQL_SERVER_CONTAINER_NAME} /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P ${SQL_SERVER_PASSWORD} -d master -Q "CREATE LOGIN no_superuser WITH PASSWORD = 'no_superuser_password', DEFAULT_DATABASE = master , CHECK_POLICY = OFF, CHECK_EXPIRATION = OFF"
```

Comment:
This is the main fix of this PR. We have to use the username `sa` instead of `SA`. This is because SQL Server treats username with case-sensitive if we set collation = `Japanese_BIN2`.